### PR TITLE
Added the frontmatter column to meta

### PR DESCRIPTION
**Pull request by tpatel**
issue https://github.com/TryGhost/Team/issues/452

I've just added a new column to the `post_meta` table (future `meta` table) to prepare for adding a frontmatter feature to posts.

### CODE REVIEW

File: core/server/data/migrations/versions/4.0/15-add-frontmatter-column-to-meta.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const {createAddColumnMigration} = require('../../utils');
+
+module.exports = createAddColumnMigration('posts_meta', 'frontmatter', {
+    type: 'text',
+    maxlength: 65535,
+    nullable: false
```
Comment (daniellockyer) [inline core/server/data/migrations/versions/4.0/15-add-frontmatter-column-to-meta.js:6]: `nullable: false` vs `nullable: true` in schema definition 🙂

Comment (tpatel) [reply core/server/data/migrations/versions/4.0/15-add-frontmatter-column-to-meta.js:6]: good catch 🙊

Comment (tpatel) [reply core/server/data/migrations/versions/4.0/15-add-frontmatter-column-to-meta.js]: fixed ✔️

**Review by naz — APPROVED** [review]
:rocket:

**Review by daniellockyer — APPROVED** [review]

**Review by allouis — APPROVED** [review]

**Pull request merged**